components/compositing/compositor.rs

```diff
@@ -358,26 +306,9 @@ fn initialize_png(width: usize, height: usize) -> RenderTargetInfo {
 
     gl::bind_texture(gl::TEXTURE_2D, 0);
 
-    let renderbuffer_ids = if opts::get().use_webrender {
```

Comment:
That piece of code was WR-only, but it's now not required by WR at all - so I've removed it completely.